webpack.config.js

```diff
@@ -105,6 +105,12 @@ const webpackConfig = {
 		chunkFilename: '[name].[chunkhash].min.js', // ditto
 		devtoolModuleFilenameTemplate: 'app:///[resource-path]',
 	},
+	optimization: {
+		splitChunks: {
```

Comment:
we also want `optimization.splitChunks.name: false`

Reply:
maybe. it looks like things moved back to an id-based chunk id, which won't be stable (likely) across builds.

Reply:
if we don't get `[name]` out of the split chunks, then even just a chunk using a new vendor module can rename that vendor chunk.